src/app/patient-summary/components/PatientSummary.tsx

```diff
@@ -15,9 +16,14 @@ export default function PatientSummary({ fhirBundle }) {
     dataTabs.map((resourceType) => {
       return (
         <TabPanel value={resourceType} index={selectedTab} key={resourceType}>
-          <div>
-            Data of <b>{resourceType}</b> will be displayed here.
-          </div>
+          {/* TODO: Consider enhancing the conditional rendering of resources */}
+          {resourceType === "Patient" ? (
+            <Patient data={fhirBundle} />
```

Comment:
Question/Suggestion: Wondering if we should just be sending the data the component will be needing. In this case, we want to display the Patient resource data, but not all the other data in the bundle.

Wondering if there might be a case where some data from another resource is shown in a specific resource type component?

Maybe something you already working on in a different PR, but maybe consider refactoring the conditional logic operation to handle a range of resourceTypes?

Reply:
Hello @BMartinos 

- Yes, we could streamline the process by sending only the necessary parts of the bundle to each component. 

I've been selecting the specific parts of the bundle within the component implementation, but I believe it would be more efficient to move this filtering logic to the parent component.

- Currently, the Patient Component doesn't require external resources, so it's not an issue. However, I’m aware that some resources, like the Composition resource, depend on data from other resource types. We can adjust the filtering logic depending on the resource type we're working with.